mac

```diff
@@ -89,132 +117,192 @@ gem_install_or_update() {
-fi
+      append_to_zshrc '# recommended by brew doctor'
 
-fancy_echo "Updating Homebrew formulae ..."
-brew update --force # https://github.com/Homebrew/brew/issues/1151
-brew bundle --file=- <<EOF
-tap "homebrew/services"
-tap "universal-ctags/universal-ctags"
-tap "heroku/brew"
+      # shellcheck disable=SC2016
+      append_to_zshrc 'export PATH="/usr/local/bin:$PATH"' 1
 
-# Unix
-brew "universal-ctags", args: ["HEAD"]
-brew "git"
-brew "openssl"
-brew "reattach-to-user-namespace"
-brew "the_silver_searcher"
-brew "tmux"
-brew "vim"
-brew "watchman"
-brew "gpg2"
-brew "zsh"
+      export PATH="/usr/local/bin:$PATH"
+  fi
 
-# Heroku
-brew "heroku/brew/heroku"
+  sudo chown -R $(whoami) $(brew --prefix)/*
 
-# GitHub
-brew "hub"
+  if brew list | grep -Fq brew-cask; then
+    fancy_echo "Uninstalling old Homebrew-Cask ..."
+    brew uninstall --force brew-cask
+  fi
+}
+
+append_general_dependencies() {
+  fancy_echo "Appending general dependencies to Brewfile"
+
+  tee "$HOME/.Brewfile" <<-EOF
+    # General
+    tap "thoughtbot/formulae"
+    tap "homebrew/services"
+    tap "universal-ctags/universal-ctags"
+    tap "github/gh"
+
+    # mas-cli to install macOS apps
+    brew "mas"
+
+    # General apps
+    cask "keybase"
+    cask "slack"
+    cask "google-chrome"
+    cask "1password"
+    cask "1password-cli"
+    cask "skitch"
+    cask "postman"
+    cask "iterm2"
+
+    # Unix
+    brew "universal-ctags", args: ["HEAD"]
+    brew "git"
+    brew "openssl"
+    brew "the_silver_searcher"
+    brew "watchman"
+    brew "gpg2"
+    brew "zsh"
+
+    # GitHub CLI
+    brew "github/gh/gh"
+
+    # Editor
+    # cask "visual-studio-code"
+
+    # Programming language prerequisites and package managers
+    brew "libyaml" # should come after openssl
+    brew "coreutils"
+EOF
+}
 
-# Image manipulation
-brew "imagemagick"
+append_web_dependencies() {
+  fancy_echo "Appending web's dependencies to Brewfile"
 
-# Testing
-brew "qt@5.5" if MacOS::Xcode.installed?
+  tee -a "$HOME/.Brewfile" <<-EOF
+    # Web
+    tap "heroku/brew"
+    tap "phrase/brewed"
 
-# Programming language prerequisites and package managers
-brew "libyaml" # should come after openssl
-brew "coreutils"
-brew "yarn"
-cask "gpg-suite"
-cask "docker"
+    # Languages
+    brew "go"
+    brew "elixir"
 
-# Databases
-brew "postgres"
-brew "redis"
+    # Devops
+    cask "docker"
+    brew "heroku"
+    brew "awscli"
 
+    # Image manipulation
+    brew "imagemagick"
+
+    # Databases
+    brew "postgres"
+
+    # Others
+    brew "yarn"
+    brew "phrase"
 EOF
+}
 
-# Basics Apps
+append_ios_dependencies() {
+  fancy_echo "Appending iOS's dependencies to Brewfile"
 
-if [ ! -d "/Applications/Google Chrome.app" ]; then
-  fancy_echo "Installing Google Chrome ..."
-  brew cask install google-chrome
-  fancy_echo "Google Chrome installed."
-fi
+  tee -a "$HOME/.Brewfile" <<-EOF
+    # iOS
+    brew "cocoapods"
+    cask "figma"
+    cask "proxyman"
+    cask "sourcetree"
```

Comment:
Do we need sourcetree as default? 🤔

Reply:
@byhbt Do you need it?
It was requested by iOS chapter, so I put it only on iOS group.
I think in Web chapter, we almost use git CLI, so I think it's redundant for us

Reply:
As we moved to github probabl makes more sense to install github desktop. I had to use it when I did normal web not ruby for example

Reply:
I added Github Desktop to general group